src/LuaEngine/LuaFunctions.cpp

```diff
@@ -689,6 +689,7 @@ ElunaRegister<Player> PlayerMethods[] =
     { "ModifyMoney", &LuaPlayer::ModifyMoney },
     { "LearnSpell", &LuaPlayer::LearnSpell },
     { "LearnTalent", &LuaPlayer::LearnTalent },
+    {"SetGlyph",&LuaPlayer::SetGlyph},
```

Comment:
Please adjust the spacing next to brackets to match the code style.

Reply:
Should be resolved. Thanks @r-o-b-o-t-o for the directly applicable suggestions.
@55Honey is there a coding style reference somewhere?

Reply:
Nothing is written explicitely. We stick with the design of the existing blueprints.